<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>empapplications</class>
 <widget class="QDialog" name="empapplications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twOrg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teResult">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFIO">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePhone">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEmail">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCust">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Заказчик</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFind">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
